v2/googlecloud-to-googlecloud/src/main/resources/bigtable-changestreams-to-vector-embeddings-command-spec.json

```diff
@@ -0,0 +1,7 @@
+{
+  "mainClass": "com.google.cloud.teleport.v2.templates.BigtableChangeStreamsToVectorSearch",
```

Comment:
I have no idea what the purpose of this file is, I copied it directly from `bgitable-changestreams-to-gcs-command-spec.json` and did `s/gcs/vector-search`.

Reply:
I believe this is put in place to enable this template to show up in the dropdown within the google cloud console.